Hotfix/correct insertion pos

hobrien opened the pull request:
For deletions, POS should be set to the base _before_ the deletion so that the ALT allele is not an empty string. For insertions, POS is the matches the position of the insertion. This script was subtracting 1 base from POS for all indels. This PR changes this to only subtract for deletions. The easiest way to do this was to calculate pos in the function that determines the REF and ALT values. I also added an assertion to the test that checks the chrom, pos, ref, and alt of the vcf line against the corresponding values in GnomAD.

afaulconbridge reviewed (commented):
Do we also need the `pos -=1` on line 170 as well? Might be easier if thats folded in too.

hobrien commented:
I forgot to take that out 😳. I guess I still do need a -= 1 somewhere

hobrien commented:
That explains why the tests all failed the first time. I've removed it now and reverted the changes to get_alleles_for_indel. No idea what's up with the CI failure
